code/modules/admin/tabs/event_tab.dm

```diff
@@ -376,7 +376,34 @@
 	if(!check_rights(R_ADMIN))
 		return
 
-	give_medal_award()
+	give_medal_award(null, TRUE)
```

Comment:
this is unclear, what is TRUE? what is null? used named arguments, instead of specifying null